packages/block-template-custom-element/.gitignore.dist

```diff
@@ -1,2 +1,3 @@
 dist
 node_modules
+.bprc.json
```

Comment:
New lines in all three `.gitignore.dist` 🙏

Reply:
https://github.com/blockprotocol/blockprotocol/pull/575/commits/bd8dd3436a6d588d763e6211c473151178f9d134